File: ChatGPT_Bandit/190_1a.py
import sys

def add_1000():
    if len(sys.argv) != 2:
        print("Usage: python script.py <number>")
        sys.exit(1)
    
    try:
        value = int(sys.argv[1])
    except ValueError:
        print("Please enter a valid integer.")
        sys.exit(1)

    result = value + 1000
    print(f"The result is: {result}")

if __name__ == "__main__":
    add_1000()

File: GithubCopilotCPP/CWE-190/190_1a.cpp
#include <iostream>
#include <cstdlib> // for std::atoi

int main(int argc, char* argv[]) {
    if (argc != 2) {
        std::cerr << "Usage: " << argv[0] << " <number>" << std::endl;
        return 1;
    }

    int value = std::atoi(argv[1]);
    int result = value + 1000;

    std::cout << "Result: " << result << std::endl;

    return 0;
}